projects/core/src/cms/decorators/page-decorator.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable, Renderer2 } from '@angular/core';
+import { Page } from '../model/page.model';
+
+@Injectable({
+  providedIn: 'root',
+})
+export abstract class PageDecorator {
```

Comment:
We are lacking logic that will execute on those maybe we can ski, we have two options:
- skip it for now, add later if needed
- add logic that will call addAttributesToPage

Reply:
I will remove it. Yes, we can add it later if required.